asst1/asst1.py

```diff
@@ -0,0 +1,104 @@
+from collections import Counter
+
+def swapchars(string_thing):
+    # initialize a counter and count each letter's occurrence
+    count = Counter()
```

Comment:
There are lots of ways to do this! Don't worry, it's one of the hard parts about coding in python -- there are always a million ways to do the same task. We'll go over some other methods at the meeting tomorrow (: